src/services/sendLinks.ts

```
@@ -82,18 +82,66 @@ type UpdateLinkBody = {
+            requestBody = new FormData()
+
+            requestBody.append(
+                'attachment',
+                sendLink.attachment,
+                sendLink.filename || (sendLink.attachment as File).name
+            )
+
+            // append other properties from sendLink to FormData
+            // exclude 'attachment', its already handled above
+            for (const key in sendLink) {
+                if (sendLink.hasOwnProperty(key) && key !== 'attachment') {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)